### ISSUE TITLE
Feature Idea: SnapKap

### ISSUE THREAD

**Issue by onordbo** [wulkano/kap#154]
I had a thought the other day, when I was using Kap to just show a bunch of stuff quickly to my team. I kept ending up with all these out of context useless gifs laying around my computer, constantly cleaning out folders to conserve space.

And then I realised, Evernote's Skitch Screenshot app has a really neat feature for sharing screenshots instantly pretty much anywhere that does file uploads, and then discarding the file after you're done with it.

Basically it works like this:

1) Record screenshot
2) Manipulate recording as needed
3) Drag on a little `.png` icon to get a png file in your hand
4) Drop that file pretty much anywhere that does file sharing. WhatsApp, Slack, GitHub, etc.
5) ???
6) Profit

[See a GIF demonstration here](https://dl.dropboxusercontent.com/u/8005186/kap.gif).

**Comment by skllcrn** [comment]
This is such a good idea! Most of the time I don't need to save the file, all I really need to do is drop it into a chat, comment or post.

**Comment by onordbo** [comment]
🚀

**Comment by onordbo** [comment]
Updating this issue with a [prototype](https://framer.cloud/GdRnT/) I made a while back to demonstrate how the feature could work.

👉  https://framer.cloud/GdRnT 👈

**Comment by albinekb** [comment]
✅  https://github.com/albinekb/kap-draggable
![drag-n-drop](https://cloud.githubusercontent.com/assets/5027156/26577619/48fe2114-452d-11e7-9f69-089d218f85cd.gif)

together with https://github.com/wulkano/kap/pull/198 

Instant gifs ⭐️

**Comment by skllcrn** [comment]
Awesome work @albinekb! This could be combined with #126, see https://github.com/electron/electron/pull/9243

**Comment by skllcrn** [comment]
https://github.com/kevva/kap-clipboard

**Comment by skllcrn** [comment]
Plugins for both [copying to clipboard](https://github.com/kevva/kap-clipboard) and [drag and drop](https://github.com/albinekb/kap-draggable) are available in the Kap 2.0 Beta, thank you @kevva and @albinekb!
